presto-main/src/main/java/com/facebook/presto/execution/SqlQueryManager.java

```diff
@@ -174,6 +171,8 @@ public SqlQueryManager(
 
         this.clusterSizeMonitor = requireNonNull(clusterSizeMonitor, "clusterSizeMonitor is null");
```

Comment:
note this line get removed in https://github.com/prestodb/presto/pull/12801/commits/bfa2524bae27cf40a021f51922d70af05dca27ee :)